test/autoCompletion.test.ts

```diff
@@ -1488,6 +1488,39 @@ describe('Auto Completion Tests', () => {
           .then(done, done);
       });
 
+      it('Array of enum autocomplete on 2nd position without `-`', (done) => {
+        languageService.addSchema(SCHEMA_ID, {
+          type: 'object',
+          properties: {
+            references: {
+              type: 'array',
+              items: {
+                enum: ['Test'],
+              },
+            },
+          },
+        });
+        const content = 'references:\n  - Test\n  |\n|';
+        const completion = parseSetup(content);
+        completion
+          .then(function (result) {
+            assert.deepEqual(
+              result.items.map((i) => ({ label: i.label, insertText: i.insertText })),
+              [
+                {
+                  insertText: 'Test',
```

Comment:
there should be '- Test' insert text, but it's another problem

Reply:
Can you file that as an issue?

Reply:
it's only a small detail I think, but logged the issue
https://github.com/redhat-developer/yaml-language-server/issues/930